spec/facades/today_schedule_facade_spec.rb

```diff
@@ -56,76 +56,52 @@
     end
   end
 
-  describe '#holiday_assignments' do
+  describe '#ooo_assignments' do
     it 'returns only the assignments for the holiday leave type' do
       holiday_leave_type = LeaveType.new(id: 123, name: 'Holiday')
       allow(LeaveTypeFinder).to receive(:call).and_return([holiday_leave_type])
 
       holiday_assignment = Assignment.new(project_id: 123)
       stub_assignment_finder(assignments: [holiday_assignment])
 
-      result = described_class.new.holiday_assignments
+      result = described_class.new.ooo_assignments
 
       expect(result.count).to eq(1)
-      expect(result.first).to eq(holiday_assignment)
+      expect(result.first).to eq(['Holiday', [holiday_assignment]])
     end
 
-    context 'when there is no leave type' do
-      it 'returns an empty array' do
-        allow(LeaveTypeFinder).to receive(:call).and_return([])
-
-        result = described_class.new.holiday_assignments
-
-        expect(result).to eq([])
-      end
-    end
-
-    context 'when there are no assignments to holiday' do
-      it 'returns an empty array' do
-        holiday_leave_type = LeaveType.new(id: 123, name: 'Holiday')
-        allow(LeaveTypeFinder).to receive(:call).and_return([holiday_leave_type])
-        stub_assignment_finder(assignments: [])
-
-        result = described_class.new.holiday_assignments
-
-        expect(result).to eq([])
-      end
-    end
-  end
-
-  describe '#sick_assignments' do
-    it 'returns only the assignments for the holiday leave type' do
+    it 'returns only the assignments for the sickness leave type' do
```

Comment:
Perhaps these two tests could be combined now that `#ooo_assignments` handles multiple leave types?

Also, shouldn't this include assignments that *shouldn't* be included, and test that they aren't present in the result?